### DISCUSSION TITLE
救助，为什么我手机上登绳网成功了，结果文章一直刷新不出来

### DISCUSSION THREAD

**Discussion by nhb-bcd** [share121/inter-knot#2559] [其他]
<img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/35c084da0600b894ef805bf90aa57318_6886091572209519570.gif" alt="期待呜呜" width="75" >
求大佬一下

**Comment by YuTinMIn** [comment]
可能是没有加速器的加速效果不好<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by nhb-bcd** [reply]
手机怎么也下载不了Watt

**Reply by Yoimiya-M** [reply]
兄弟啊，watt是PC端的

**Reply by nhb-bcd** [reply]
哪有手机端的加速器吗

**Reply by Yoimiya-M** [reply]
安卓我知道一个起飞加速器，iOS我知道一个快喵，一般渠道获取不到，你试试看能不能从啥犄角旮旯里搜出来

**Reply by CBBC123** [reply]
推荐一个B站视频https://b23.tv/Zbafhzrhttps://b23.tv/Zbafhzr
我就是看这个解决了和你一样的问题的

**Reply by zzyuJPG** [reply]
手机其实可以下载watt

**Comment by LuAn-afl** [comment]
https://www.123pan.com/s/N3fcVv-t0jQh?提取码:aCL5     兄弟试试这个时间没了点上面的清除数据就行弹更新提示换网就行

**Comment by CBBC123** [comment]
推荐一个B站视频https://b23.tv/Zbafhzrhttps://b23.tv/Zbafhzr
我就是看这个解决了和你一样的问题的

**Reply by LuAn-afl** [reply]
> 推荐一个B站视频https://b23.tv/Zbafhzrhttps://b23.tv/Zbafhzr 我就是看这个解决了和你一样的问题的

怎么什么都没有？

**Reply by richi-shek** [reply]
> > 推荐一个B站视频https://b23.tv/Zbafhzrhttps://b23.tv/Zbafhzr 我就是看这个解决了和你一样的问题的
> 
> 怎么什么都没有？

B站视频的链接是https://b23.tv/Zbafhzr ，他给的链接(Ctrl + V ) * 2
![68747470733a2f2f75706c6f61642d6262732e6d69796f757368652e636f6d2f75706c6f61642f323032332f30332f30392f33313733626333653639666366656265636664613734313334306339616432325f363834313235303331303735363237353637382e706e67](https://github.com/user-attachments/assets/c211b39a-956b-488d-840e-0b9442d5417e)
